packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -73,6 +73,7 @@ This package contains documentation for rubygem-%{gem_name}.
 %install
 mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
+rm %{buildroot}%{gem_dir}/gems/%{gem_name}-%{version}/.babelrc
```

Comment:
Notice I'm removing Babel's `babelrc` because it's only needed to run `npm run test` on the plugin. But if you try to compile the bundle with it in production, it fails because the `foreman_remote_execution` gem folder does not have - it can't have - a `node_modules` folder. In development it's all fine.

Reply:
I think we should exclude this from the actual gem.

Reply:
This is equal to `rm %{buildroot}%{gem_instdir}/.babelrc` and we already have `%exclude %{gem_instdir}/.*` in `%files`. Does this line actually make a difference? If it does, I'd prefer to use the `%{gem_instdir}` macro instead of writing it out but ideally we'd remove this line.